### PR TITLE
Fixed a few typos in README.md

### PR DESCRIPTION
**Pull request by trunapushpa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,15 +12,15 @@ Here is video of a race (running code from students):
...
+The race cars have to recognize the race track, the obstacles, and the bonus areas
+calculate the best path to take to avoid the pitfalls and collect bonus points.
 The cars move autonomously on the screen within the race track game with no interference
 from the students. No joystick or mouse would be used.
  
 In order to control the car movements, the students needs to implement a 'driver'.
 This code is controlling the car and will decide what will be the next action of the car.
 
-For each type of obstacles there is a different action, and different points.
+Each type of obstacle has a different action, and different points.
```
Comment (nirs) [inline README.md]: Looks good.

File: README.md [hunk 2]
```diff
@@ -12,15 +12,15 @@ Here is video of a race (running code from students):
 alt="ROSE Race Car Game" width="860" height="720" border="10" /></a>
 
 In this game, two race cars compete to achieve the most points.
-The race car have to recognize the race track, the obstacles, and the bonus areas;
-calculate the best path to take to avoid the pitfalls; and collect bonus points.
+The race cars have to recognize the race track, the obstacles, and the bonus areas
```
Comment (nirs) [inline README.md]: Every student program single race car, so I don't think it helps to talk about "cars" here.

**Comment by nirs** [comment]
@trunapushpa can you remove the unneeded change and rebase on master?

**Comment by rollandf** [comment]
@trunapushpa Thanks!

**Pull request merged**